### ISSUE TITLE
[bug] POST samples HTTP API not working

### ISSUE THREAD

**Issue by javimoya** [wit-ai/wit#700]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
bug

**What is the current behavior?**
POST samples on HTTP API not working

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
curl -XPOST 'https://api.wit.ai/samples?v=20170307' \
  -H "Authorization: Bearer JQNADWOL6GKYL4K4RHLBFP3V4EEKT3T2" \
  -H "Content-Type: application/json" \
  -d '[{
        "text": "I really want an appointment",
        "entities": [
          {
            "entity": "intent",
            "value": "appt_make"
          }
        ]
      }]'

**What is the expected behavior?**
A new expression sample is added in the intent (already previosly created)

**What is the App ID where you are experiencing this issue (if applicable)?**
all of them
example: 59879b95-fd0c-4c52-821f-3f5bf055a1db

**Comment by l5t** [comment]
cc @stopachka

**Comment by patapizza** [comment]
Hi @javimoya,

As specified in the [docs](https://wit.ai/docs/http/20170307#post--samples-link), the entities must exist before using POST /samples. You can first create the entities using POST /entities.

Thanks!

**Comment by zhuang13** [comment]
@patapizza  I have the same problem. notice the problem:

What is the expected behavior?
A new expression sample is added in the intent **(already previosly created)**

**Comment by stopachka** [comment]
Hey @zhuang13 samples is currently experiencing issues, we're working on unblocking asap 

ref: https://github.com/wit-ai/wit/issues/699

**Comment by Dierme** [comment]
Hey! I started experiencing the same issue.
Firstly, i create new entities by 

```
curl -XPOST 'https://api.wit.ai/entities?v=20170307' \
  -H "Authorization: Bearer XEXPZR2CLQQIXLS7ZS4CMWUNZM7NIERE" \
  -H "Content-Type: application/json" \
  -d '{"doc":"A city that I like",
       "id":"favorite_city"}'

```

After that, i try to push new samples in.

```
curl -XPOST 'https://api.wit.ai/samples?v=20170307' \
  -H "Authorization: Bearer XEXPZR2CLQQIXLS7ZS4CMWUNZM7NIERE" \
  -H "Content-Type: application/json" \
  -d '[{
        "text": "I want to fly to sfo",
        "entities": [
          {
            "entity": "favorite_city",
            "value": "London"
          }
        ]
      }]'

```

And samples are not getting created. It was working a week ago, but stopped a few days ago. My app ID is 5a75df02-4a2f-454a-8a0b-bfabaad0ae3c

**Comment by blandinw** [comment]
@Dierme please create a new issue next time, since this one is pretty old, it's unlikely to be the same root cause. How long did you wait for the samples to be created? Can you confirm everything is working fine now?
